### PR TITLE
some minor typos/clarifications

### PR DESCRIPTION
**Pull request by paulk-asert**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: scala-package/README.md [hunk 1]
```diff
@@ -156,21 +156,21 @@ mvn -Dsuites=org.apache.mxnet.NDArraySuite integration-test
...
+Also it produces the native library in `native/target`, which you can use in conjunction with the `core` module.
 
 Deploy to repository
 --------------------
 
-By default, `maven deploy` will deploy artifacts to local file system, you can file then in: ``scala-package/deploy/target/repo`` folder.
+By default, `maven deploy` will deploy artifacts to local file system, you can find them in the ``scala-package/deploy/target/repo`` folder.
 
-For nightly build in CI, a snapshot build will be uploaded to apache repository with follow command:
+For nightly builds (typically done by CI), a snapshot build will be uploaded to an apache snapshot repository with the following command:
```
Comment (ChaiBapchya) [inline scala-package/README.md]:
```suggestion
For nightly builds (typically done by CI), a snapshot build will be uploaded to an Apache snapshot repository with the following command:
```

File: scala-package/README.md [hunk 2]
```diff
@@ -156,21 +156,21 @@ mvn -Dsuites=org.apache.mxnet.NDArraySuite integration-test
...
 
 ```bash
 cd scala-package
 mvn deploy -Pnightly
 ```
 
-Use following command to deploy release build (push artifacts to apache staging repository):
+Use the following command when performing a release (pushes artifacts to an apache staging repository):
```
Comment (ChaiBapchya) [inline scala-package/README.md]: Previous statement was good enough, I guess (in terms of comprehensibility)
.
```suggestion
Use the following command when performing a release (pushes artifacts to an Apache staging repository):
```

File: scala-package/README.md [hunk 3]
```diff
@@ -156,21 +156,21 @@ mvn -Dsuites=org.apache.mxnet.NDArraySuite integration-test
...
 
 If everything goes well, you will find jars for `assembly`, `core` and `example` modules.
-Also it produces the native library in `native/target`, which you can use to cooperate with the `core` module.
+Also it produces the native library in `native/target`, which you can use in conjunction with the `core` module.
 
 Deploy to repository
 --------------------
 
-By default, `maven deploy` will deploy artifacts to local file system, you can file then in: ``scala-package/deploy/target/repo`` folder.
+By default, `maven deploy` will deploy artifacts to local file system, you can find them in the ``scala-package/deploy/target/repo`` folder.
```
Comment (ChaiBapchya) [inline scala-package/README.md]:
```suggestion
By default, `maven deploy` will deploy artifacts to local file system. You can find them in the ``scala-package/deploy/target/repo`` folder.
```

File: scala-package/README.md [hunk 4]
```diff
@@ -133,8 +133,8 @@ Also, add the dependency which corresponds to your platform to the ```dependenci
 Build From Source
 -----------------
 
-Checkout the [Installation Guide](http://mxnet.incubator.apache.org/install/index.html) contains instructions to install mxnet package and build it from source. Scala maven build assume you already have a ``lib/libmxnet.so`` file.
-If you have built MXNet from source and are looking to setup Scala from that point, you may simply run the following from the MXNet source root, Scala build will detect your platform (OSX/Linux) and libmxnet.so flavor (CPU/GPU):
+The [Installation Guide](http://mxnet.incubator.apache.org/install/index.html) contains instructions to install mxnet or build it from source. The Scala/Java package is built from source using Maven. The maven build assumes you already have a ``lib/libmxnet.so`` file.
```
Comment (ChaiBapchya) [inline scala-package/README.md]:
```suggestion
The [Installation Guide](http://mxnet.incubator.apache.org/install/index.html) contains instructions to install Mxnet or build it from source. The Scala/Java package is built from source using Maven. The maven build assumes you already have a ``lib/libmxnet.so`` file.
```

**Comment by ChaiBapchya** [comment]
@mxnet-label-bot add [Doc, Scala]

**Comment by lanking520** [comment]
@aaronmarkham

**Pull request merged**